apps/web/app/(app)/environments/[environmentId]/integrations/airtable/components/AddIntegrationModal.tsx

```diff
@@ -291,7 +291,7 @@ export const AddIntegrationModal = ({
               <div className="space-y-4">
                 <div>
                   <Label htmlFor="Surveys">Questions</Label>
-                  <div className="mt-1 rounded-lg border border-slate-200">
+                  <div className="mt-1 max-h-[350px] overflow-y-scroll rounded-lg border border-slate-200">
```

Comment:
We can change `overflow-y-scroll` to `overflow-y-auto`,so that is shows the scroll bar only when required and not for less number of questions 😊

<img width="703" alt="Screenshot 2024-06-26 at 10 10 45 AM" src="https://github.com/formbricks/formbricks/assets/67850763/62b94826-0060-4b27-9972-d156e33b32a2">